integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMonitoringExporter.java

```diff
@@ -1368,35 +1290,19 @@ public static String createPushImage(String dockerFileDir, String baseImageName,
 
   //download src from monitoring exporter github project and build webapp.
   private static void installMonitoringExporter() {
-    logger.info("create a staging location for monitoring exporter github");
     Path monitoringTemp = Paths.get(RESULTS_ROOT, "monitoringexp", "srcdir");
-    assertDoesNotThrow(() -> FileUtils.deleteDirectory(monitoringTemp.toFile()));
-    assertDoesNotThrow(() -> Files.createDirectories(monitoringTemp));
+    monitoringExporterSrcDir = monitoringTemp.toString();
+    cloneMonitoringExporter(monitoringExporterSrcDir);
     Path monitoringApp = Paths.get(RESULTS_ROOT, "monitoringexp", "apps");
-    assertDoesNotThrow(() -> FileUtils.deleteDirectory(monitoringApp.toFile()));
+    assertDoesNotThrow(() -> org.apache.commons.io.FileUtils.deleteDirectory(monitoringApp.toFile()));
     assertDoesNotThrow(() -> Files.createDirectories(monitoringApp));
     Path monitoringAppNoRestPort = Paths.get(RESULTS_ROOT, "monitoringexp", "apps", "norestport");
-    assertDoesNotThrow(() -> FileUtils.deleteDirectory(monitoringAppNoRestPort.toFile()));
+    assertDoesNotThrow(() -> org.apache.commons.io.FileUtils.deleteDirectory(monitoringAppNoRestPort.toFile()));
```

Comment:
you can static import org.apache.commons.io.FileUtils.deleteDirectory

Reply:
fixed